cassandra-config/pom.xml

```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.lightbend.lagom.sample.chirper</groupId>
+        <artifactId>activator-lagom-java-chirper</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>cassandra-config</artifactId>
```

Comment:
Thanks for the catch. I got the `cassandra-config` copied from the Maven archetype, but it's not required. I've tested that signup, login, logout, and chirps are working despite `cassandra-config` being removed.